sysl2/sysl/Parser_test.go

```diff
@@ -70,36 +78,71 @@ func parseComparable(filename, root string) (*sysl.Module, error) {
 	return module, nil
 }
 
-func parseAndCompare(filename, root string) (bool, error) {
-	goModule, err := parseComparable(filename, root)
+func parseAndCompare(filename, root, golden string, goldenModule *sysl.Module) (bool, error) {
+	module, err := parseComparable(filename, root)
```

Comment:
Not part of this PR, but seeing as it's a cleanup: remove line 69 above (`// app.SourceContext.Start.Col = 0`)